xml/Microsoft.CLRAdmin/IDataObject.xml

```diff
@@ -178,10 +209,22 @@
         <Parameter Name="b" Type="Microsoft.CLRAdmin.STGMEDIUM" RefType="ref" />
       </Parameters>
       <Docs>
-        <param name="pFormatEtc">To be added.</param>
-        <param name="b">To be added.</param>
+        <param name="pFormatEtc">A pointer to the <see cref="T:Microsoft.CLRAdmin.FORMATETC" /> defining the format, medium, and target device for calls to retrieve data.</param>
+        <param name="b">A pointer to the <see cref="T:Microsoft.CLRAdmin.STGMEDIUM" /> indicating the storage medium containing the returned data.</param>
         <summary>This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
```

Comment:
All the APIs that have this sentence in this PR need to have `<forInternalUseOnly />` added after the remarks and have the summary replaced with some real description. Can you please do that throughout the rest of this PR? Otherwise, I'll soon hit the suggestions limit.